### ISSUE TITLE
[AQ40] Twin emperors door open without killing Huhuran

### ISSUE THREAD

**Issue by BeefSteaker** [azerothcore/azerothcore-wotlk#13395]
original issue https://github.com/chromiecraft/chromiecraft/issues/4174
### Current Behaviour

Twin emperors door is open without killing Huhuran, allowing you to skip this boss

![image](https://user-images.githubusercontent.com/11332559/195372528-a7a5dfba-0f5e-47b0-a542-3a25f4f46458.png)

### Expected Blizzlike Behaviour

Princess Huhuran shouldnt be skipable (mandatory boss!). Gate to Twins should be closed or C'Thun not accessible if Huhuran is still alive.

### Source

https://classic.wowhead.com/guides/temple-ahnqiraj-aq40-overview-classic-wow

![image](https://user-images.githubusercontent.com/11332559/195371303-860b1110-f078-4a66-a278-f1c55ee831e8.png)
https://classic.wowhead.com/npc=15509/princess-huhuran

Princess Huhuran wouldnt be listed everywhere as a mandatory boss if they were skippable

### Steps to reproduce the problem

.tele aq40
.go xyz -8912 1358 -95

### Extra Notes

Princess Huhuran is a major gear check for aq40 as it requires Nature resistance gear. it being skippable eliminates a significant challenge of the raid, and allows access to c'thun/ouro loot earlier than might be expected

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/77faa1d84d85199170ef540a43b5ed824de43702

### Operating system

Ubuntu 20.04

### Custom changes or Modules

[mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
    [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
    [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
    [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
    [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
    [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
    [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
    [mod-dmf-switch](https://github.com/azerothcore/mod-dmf-switch)
    [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
    [mod-eluna](https://github.com/azerothcore/mod-eluna)
    [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
    [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
    [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
    [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
    [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
    [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
    [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
    [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
    [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
    [mod-transmog](https://github.com/azerothcore/mod-transmog)
    [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
    [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
    [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
    [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
    [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
    [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
    [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
    [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
    [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
    [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)
    [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

**Comment by Gultask** [comment]
Is this still valid in WotLK?

**Comment by BeefSteaker** [comment]
Hmm, I'll look into it. 
Other doors like after skeram & after twin emperor's are still closed like they used to be

**Comment by Gultask** [comment]
Yes, we fixed that not too long ago :)
When I ran AQ40 the entrance to Twin Emps was always open, but I did check it in retail and not WotLK.

**Comment by Nyeriah** [comment]
> Yes, we fixed that not too long ago :) When I ran AQ40 the entrance to Twin Emps was always open, but I did check it in retail and not WotLK.

This is what I remember as well, it is always open. Huhuran aggros if you try to skip past her iirc.

**Comment by Gultask** [comment]
I don't remember an areatrigger but I can check

**Comment by Gultask** [comment]
Nope, no areatriggers, and no linking, either. Huhuran is skippable in 9.2.7

**Comment by TheSCREWEDSoftware** [comment]
[How to Solo AQ40 (The Temple of Ahn'Qiraj) - Speed Run for Gold and Transmogrification Items](https://www.youtube.com/watch?v=Ypbi9vF15uM&ab_channel=warcraftprofessor) (WOTLK)

and

[26:43min #1 AQ40 Speedrun by NOTA in P5](https://www.youtube.com/watch?v=8A-lVix1zM0&ab_channel=Sceto) (Vanilla)

and

[SALAD BAKERS - AQ40 World record speedrun 29min 37sec](https://www.youtube.com/watch?v=RF1hE_YFx4g&ab_channel=Nharz) (Vanilla)

I can't find anyone or guild speedruning AQ20 or 40 in Wrath, but if none of the videos (im aware the first one the guy is doing a transmog run) don't seem to skip the boss when trying to a beat timer, I would say the doors should be closed.
